### DISCUSSION TITLE
What to fill in the port number in the game？

### DISCUSSION THREAD

**Discussion by TC999** [bedrock-tool/bedrocktool#27] [Q&A] [answered]
*

**Comment by olebeck** [comment, marked as answer]
Port 19132 as it says in the output and is the default for the game

**Reply by TC999** [reply]
<img width="474" alt="捕获" src="https://user-images.githubusercontent.com/88823709/213843054-fefbc8df-a878-4b0f-8b1a-30596aa650fe.PNG">
世界に接続できませんでした
サーバポートは19130…

**Reply by olebeck** [reply]
are you using the netease version of minecraft bedrock, that version doesnt support regular bedrock servers since they have made some changes to the protocol. supporting netease clients may be something i can look into implementing in the future

**Reply by olebeck** [reply]
or actually the issue could be https://github.com/bedrock-tool/bedrocktool/discussions/28#discussioncomment-4744913
